### ISSUE TITLE
OTR: Align GO and Reactome on ciliary processes

### ISSUE THREAD

**Issue by paolaroncaglia** [geneontology/go-ontology#12487]
@JohnvanDam @krchristie @deustp01

As part of the cilia GO project, we need to make sure that GO and Reactome align on representation of ciliary processes.
Reactome recently worked on this area, this is the overall pathway we should look at:

Assembly of the primary cilium (R-HSA-5617833) [Homo sapiens]
http://www.reactome.org/content/detail/R-HSA-5617833
http://www.reactome.org/PathwayBrowser/#/R-HSA-5617833&PATH=R-HSA-1852241

Issues we need to address first:

The Reactome pathway refers explicitly to the primary cilium in its name. The summation quotes, more specifically, the non-motile primary cilium. 
We need to tie this alignment work with our planned re-naming of cilia types, see https://github.com/geneontology/go-ontology/issues/12462
Which of these 4 suggested classes does the Reactome pathway cover?

1) 9+0 non-motile cilia (which include authentic primary cilia, but are not limited to them), 
2) 9+0 motile cilia (which include classic nodal cilia, but are not limited to them), 
3) 9+2 non-motile cilia, 
4) 9+2 motile cilia (which include so-called flagella and conventional motile cilia).

Karen Rothfels at Reactome authored the entry on ‘Assembly of the primary cilium’, so we may wish to consult with her here. In the meantime, Peter, please let us know if you have any comments, thanks. 

GO currently has terms for 

motile cilium
primary cilium
- motile primary cilium
- nonmotile primary cilium

and for 

cilium assembly
- motile cilium assembly
- motile primary cilium assembly
- nonmotile primary cilium assembly

We may need to merge some terms (both CC and BP) and we need to make sure what can correctly merge into what/correctly rehouse annotations if necessary.
Number of _direct, manual experimental_ annotations to BP terms:

cilium assembly [499 for 355 proteins]
- motile cilium assembly [37 for 34 proteins]
- motile primary cilium assembly [5 for 5 proteins]
- nonmotile primary cilium assembly [150 for 96 proteins]

Thanks,
Paola

**Comment by paolaroncaglia** [comment]
Note for self:
I'll need to suggest Reactome2GO mappings, and add the inverse.

**Comment by JohnvanDam** [comment]
There are many parts of that pathway we could implement. It could definitely function as a template. I do think that most, if not all, processes mentioned at Reactome are applicable to all four cilia subtypes. Biggest differences would be in the specific construction of the axonemes (9+0 motile cilium would have something akin to "assembly of motile apparatus", but not "assembly of central pair" etc.). I have the whole week blocked to work on GO except tomorrow. I will start looking into this in detail on wednesday.

**Comment by paolaroncaglia** [comment]
Thanks @JohnvanDam

**Comment by paolaroncaglia** [comment]
Notes for self, in the context of ciliary processes we should do some work on:

1) A reminder that the overall ciliary BP branch we should look at here is ‘cilium organization’. 
Sub-processes comprise ‘cilium assembly’ (specific object of this ticket), with its part ‘axoneme assembly’; and ’cilium disassembly’.
A related issue pertains to ‘cilium morphogenesis’, for which we have a dedicated ticket (https://github.com/geneontology/go-ontology/issues/12236).
Remember to look at all regulation terms as well.

2) The other branch we should focus on is ‘cilium movement’. This should be taken care of by these existing tickets:
https://github.com/geneontology/go-ontology/issues/10898
https://github.com/geneontology/go-ontology/issues/10905
https://github.com/geneontology/go-ontology/issues/11831

3) (Minor note) The existing ‘ciliary receptor clustering involved in smoothened signaling pathway’ (“Grouping of smoothened or patched receptors in a cilium, contributing to the smoothened signaling pathway.”) could be given a link occurs_in ‘cilium’.

**Comment by paolaroncaglia** [comment]
Note for self: followed up with Peter and Karen R today (by email).

**Comment by KarenRothfels** [comment]
Sorry for delay.  The Reactome pathway *technically describes formation of a 9+0 non motile primary cilium, but in practice only in as much as that is what it says it describes (in text summation form).  The pathway events don't actually describe axoneme formation in enough detail to allow them to distinguish between the four classes of cilium described above. In the GO world, is it preferable to have potentially common events labeled with a too specific term (events labeled with '9+0 non-motile primary cilium', even if they also happen in, say, 9+2 motile cilia) or to have them labeled as applying to multiple cilia types, even under a pathway heading that is more specific, as it is in this case?

**Comment by deustp01** [comment]
So I guess that where we stand on this is that 9 + 0 primary cilia can be motile or non-motile. We (Karen R) have annotated events involved in the assembly of 9 + 0 non-motile cilia but not in a way that explicitly provides a molecular feature that could be incorporated into a definition to distinguish this process from assembly of 9 + 0 motile cilia.

If that's all true, then we should associate the parent biological_process term  GO:0042384 cilium assembly with the Reactome pathway so as not to generate GAF assertions that go beyond the functions of the proteins we've actually annotated.

Meanwhile, it's probably worth some more discussion, if people have time for it, to try to identify the data needed to turn those text assertions into annotations of proteins - or has that already been tried and has it failed?

**Comment by krchristie** [comment]
I'm at TAGC this week, through Sunday 17th and won't have time to comment till I'm back to work on the 18th.

-Karen C

**Comment by JohnvanDam** [comment]
Silly question: what is a GAF assertion exactly?
I am unaware of any notable, published differences between motile and non-motile 9+0, other than that the motile version has "more" visible protein structures in it (e.g. inner and outer dynein arms) making all assignments to 9+0 immotile cilia in principle applicable to 9+0 motile cilia as well. However I am currently working on a revamp of the BP cilium assembly branch which might make matching the reactome  pathway to GO a lot easier. I do like to make the point though that the reactome pathway seems rather specifically named while most if not all the processes are applicable to all cilia types including 9+2.

**Comment by KarenRothfels** [comment]
I came to ciliary annotations from Hedgehog signaling, so I had primary non-motile cilia on the brain.
If the goal is to have GO terms and Reactome annotations align, there are 2 possibilities for changing the Reactome pathway to fit the models under discussion here:

1. add more molecular detail to the Reactome pathway to make it describe what it says it does (a 9+0 non motile cilium) - I am happy to do this if the literature exists. The pathway as is had 2 external experts, neither of whom made this suggestion, so I would have to do some digging. Or maybe someone on this thread knows whether there are molecular events that can be annotated that would make the Reactome pathway more specific.
2. change the pathway name and some summations so that the Reactome pathway can be more accurately labeled with all four cilia classes or with a more generic ciliary GO term.

**Comment by krchristie** [comment]
Similarly to John, as far as I am aware, the 9+0 motile cilia and 9+2 motile cilia have more proteins/structures than 9+0 nonmotile cilia. [I can't say anything about 9+2 non-motile cilia. I've never encountered those in my curation or reading. The only place I've heard of those was in the review that we discussed with respect to classification of cilia types.] Correspondingly, assembly of 9+0 motile cilia and 9+2 motile cilia include all the parts of assembly of 9+0 non-motile cilia, plus some additional steps to assemble structures not present in 9+0 non-motile cilia.

Then, within any class of cilia, there are specialized types. Kinocilia and the connecting cilium of photoreceptors are specialized types of 9+0 non-motile cilia. Sperm flagella are specialized 9+2 motile cilia. My impression is that assembly of these specialized cilia include all the steps of assembly of 9+0 non-motile cilia, with additional steps for assembly of additional specialized structures.

Regarding Karen R's two possibilities, personally I don't know of any molecular details that would make the current Reactome pathway specific to 9+0 non-motile cilia, since as far as I'm aware everything that happens in assembly of this type of cilia happens in assembly of all the other types of cilia also. So, I'm not sure option 1 is possible. To me, option 2 seems like the way to go.

-Karen C

**Comment by JohnvanDam** [comment]
For an overview, this is panel A of a figure I am making on ciliary components and depicts the different types of axonemes and their components:
![image](https://cloud.githubusercontent.com/assets/1461306/16955775/3f3d7f52-4dd6-11e6-9ead-09567ed9a6a6.png)

**Comment by paolaroncaglia** [comment]
@JohnvanDam @krchristie @deustp01 @KarenRothfels 
Thank you all for your feedback. Summing up, it sounds like we may safely align and cross-reference the existing Reactome pathway and GO term for
Assembly of the primary cilium (R-HSA-5617833)
and
cilium assembly (GO:0042384)
provided that we “change the pathway name and some summations so that the Reactome pathway can be more accurately labeled with all four cilia classes or with a more generic ciliary GO term”, as Karen R suggests in her option 2.

Then, if necessary, we may create more granular terms such as ‘9+0 motile cilia assembly’. However, this sounds like an optional, non-priority step to me at the moment.

Karen R, before we proceed, do you think you could check with the experts who contributed to Assembly of the primary cilium (R-HSA-5617833) please, if they agree with your option 2?

Thanks,
Paola

**Comment by KarenRothfels** [comment]
Hi again -

This module had 2 reviewers.  I have already consulted with one and he felt the Reactome pathway as described could also apply to the other cilia types as well.   I will check with the other reviewer to make sure that he concurs.

Thanks,

Karen


Karen Rothfels, PhD
Scientific Associate, Reactome

Ontario Institute for Cancer Research
MaRS Centre
661 University Avenue
Suite 510
Toronto, Ontario
Canada M5G 0A3

Tel:         647-259-4250
Email:   karen.rothfels@oicr.on.ca
Toll-free: 1-866-678-6427
Twitter: @OICR_news

www.oicr.on.ca<http://www.oicr.on.ca>

This message and any attachments may contain confidential and/or privileged information for the sole use of the intended recipient. Any review or distribution by anyone other than the person for whom it was originally intended is strictly prohibited. If you have received this message in error, please contact the sender and delete all copies. Opinions, conclusions or other information contained in this message may not be that of the organization.

**Comment by paolaroncaglia** [comment]
Thanks @KarenRothfels !

**Comment by paolaroncaglia** [comment]
Notes for self:

- [ ] Wrt linking ‘ciliary vesicle’ and ‘ciliary pocket’ (stemming from https://github.com/geneontology/go-ontology/issues/12563), this would probably fit better at the biological process level. We could refer to the Reactome entry for ‘Anchoring of the basal body to the plasma membrane’, and evaluate appropriateness/need for adding links such as ‘cilium assembly’ starts_with/has_part ‘ciliary vesicle assembly’, ‘ciliary pocket assembly’, ‘ciliary basal body docking’. 

- [ ] Once the revised GO ‘cilium assembly’ branch is in place, we should ask Karen to check with her colleague from the Chlamydomonas community to please take a look and let us know if we should restrict any terms beneath ‘cilium assembly’ to specific taxonomic groups. Currently, ‘cilium assembly’ is labelled as occurring only in eukaryotes but never in Ascomycetes; we could certainly add more constraints if needed, to it and/or to any of its children.

**Comment by paolaroncaglia** [comment]
Hi @KarenRothfels  and @deustp01 ,

Following up this, we had resolved to

“safely align and cross-reference the existing Reactome pathway and GO term for 
Assembly of the primary cilium (R-HSA-5617833)
and
cilium assembly (GO:0042384)
[where GO ‘cilium’ covers all cilia types regardless of axonemal structure and motility]
provided that we “change the pathway name and some summations so that the Reactome pathway can be more accurately labeled with all four cilia classes or with a more generic ciliary GO term”.
Then, if necessary, we may create more granular terms such as ‘9+0 motile cilia assembly’. However, this sounds like an optional, non-priority step to me [Paola] at the moment.”

@KarenRothfels checked with the experts who contributed to Assembly of the primary cilium (R-HSA-5617833) if they agree with this option, and noted in her last comment:

“This module had 2 reviewers. I have already consulted with one and he felt the Reactome pathway as described could also apply to the other cilia types as well. I will check with the other reviewer to make sure that he concurs.” 

Did the second reviewer agree please? Can we go ahead with the plan? 
Depending on the answer, we will then proceed to examine the alignment between Reactome’s subprocesses in entry R-HSA-5617833 and GO. 
 
Thanks!

Paola

**Comment by KarenRothfels** [comment]
second expert (Esben Lorentzen) weighs in:

"Dear Karen,
Thanks for your email, just back from vacation. I understand your problems with classification of types and assembly. In the literature researchers typically operates with two types of cilia, the motile 9+2 cilium and the (typically non-motile) 9+0 primary cilium. However, this is not a strict division as 9+0 cilia have also been demonstrated to have motility (nodal cilia ) and motile cilia are also sensory. Basically, the issue of motile vs. non-motile boils down to whether the MT-based axoneme of of the cilium  harbours motility factors (Outer and inner dynein arms, radial spokes, dynein-nexin complex etc).There is not too much known about how a cell decides to add motility factors to a particular  cilium, also transport of outer dynein arms have been described to rely on ODA16 and IFT46 by George witman and also the Mitchell labs.

With respect to assembly it is clear that all cilia share a common axonemal core structure composed of 9 doublets of MTs. There thus has to be a common general way to transport and assemble tubulin in all cilia (we have described this in Bhogaraju et al 2013 science, Bhogaraju et al.,  Bioassay 2014, Taschner et al., EMBO J 2016).  With respect to sensory cilia it is clear that different functions require the transport of different receptors to the ciliary membrane.  The photoreceptor cell will needs opsins in the ciliary membrane to sense light....olfactory cilia will need receptors to detect specific 'smell molecules' etc. There are thus bound to be differences between the assembly of cilia on different cells/tissue to reflect these different functionalities although the molecular basis have in most cases not been worked out in any detail.

In my view, as long as your classification contains motile cilia and primary cilia it is fine. Of course one can then make a number of sub-divisions depending on organism or cell type but I am not sure you want to get into this....

hope this helps a little bit..

best

esben

**Comment by paolaroncaglia** [comment]
Many thanks @KarenRothfels . 
If we align
Assembly of the primary cilium (R-HSA-5617833)
and
cilium assembly (GO:0042384)
[where GO ‘cilium’ covers all cilia types regardless of axonemal structure and motility]
this _will_ include motile cilia and primary cilia - as well as all other types of cilia. It seems to me that the current level at which Reactome is representing cilium assembly sub-processes (cargo trafficking etc - see a drill-down of http://www.reactome.org/PathwayBrowser/#/R-HSA-5617833&PATH=R-HSA-1852241) would still work for all types of cilia. 

@krchristie , @JohnvanDam , I think you agreed on this 'vision' earlier on, so I'll assume you still do... but please shout if not. Thanks!

**Comment by krchristie** [comment]
Yes @paolaroncaglia this works for me. Thanks!

**Comment by paolaroncaglia** [comment]
@KarenRothfels @deustp01 

Summing up, here are the action items, from the top down:

- [ ] Reactome to change the pathway name for R-HSA-5617833.2 from “Assembly of the primary cilium” to “Assembly of the cilium” or similar, and edit some summations accordingly (including those of sub-pathways). This suggested change is ready for implementation.

- [ ] Reactome to add mapping for R-HSA-5617833.2: “Represents GO Biological Process” = GO ‘cilium assembly’. Note, the current ID of GO ‘cilium assembly’ will change in about a week due to a merge. Are your mappings automatically updated in similar circumstances? If not, you may wish to hold on to this edit, and I’ll let you know as soon as the merge is completed. (GO ‘cilium assembly’ won’t change its meaning after the merge - if you’re curious, see https://github.com/geneontology/go-ontology/issues/12236#issuecomment-246941622)

All the following action items are for GO to do: 

- [ ] Add Reactome xref R-HSA-5617833.2 to GO term ‘cilium assembly’. This can be done as soon as the Reactome pathway is made to point to GO ’cilium assembly’.

- [ ] Align Reactome and GO on sub-processes. This action item needs expanding - will do.

**Comment by paolaroncaglia** [comment]
Now looking at subprocesses of ‘cilium assembly’. The hierarchy under GO ‘cilium assembly’ currently looks like this:

cilium assembly
[is_a] motile cilium assembly
[is_a] non-motile cilium assembly
[part_of] axoneme assembly
[part_of] ciliary basal body docking
[part_of] vesicle targeting, trans-Golgi to periciliary membrane compartment

‘motile cilium assembly’ and ‘non-motile cilium assembly’ derive from the merges in https://github.com/geneontology/go-ontology/issues/12462 and have, respectively, 48 and 180 manual experimental annotations. We will keep these terms because we’ve resolved that the distinction between motile and non-motile cilia is worth having. Esben Lorentzen (Reactome reviewer) noted “Basically, the issue of motile vs. non-motile boils down to whether the MT-based axoneme of of the cilium harbours motility factors (Outer and inner dynein arms, radial spokes, dynein-nexin complex etc). There is not too much known about how a cell decides to add motility factors to a particular cilium, also transport of outer dynein arms have been described to rely on ODA16 and IFT46 by George witman and also the Mitchell labs.”. As further evidence emerges, having separate terms for assembly of motile vs non-motile cilia will allow to capture it with greater specificity.

@KarenRothfels @deustp01 No action item for Reactome on this one. Next for me to do,

- [ ] Look at part_of children of GO ‘cilium assembly’ and see how they should align with Reactome.

**Comment by paolaroncaglia** [comment]
Note for self: followed up with Reactome by email today.

**Comment by KarenRothfels** [comment]
Hi Paola-

First of all, apologies for my slow turn around time these days - there seems to be more to do than hours to do it in.

Yes, the proposed action steps sound good.  I will change the pathway name and the summations to make them more inclusive of all cilia types (though I propose to put in a fudging kind of sentence to say that the pathway discusses the processes largely in reference to the primary non motile cilia, to account for the bias in the literature citations).  This will go out in our next release in December.  I can wait to add the GO biological terms until the GO merge is complete, and can continue to add subprocess terms as the mappings become clear.

Karen

**Comment by paolaroncaglia** [comment]
Hi @KarenRothfels ,

Many thanks for your feedback! 
I agree with your suggested wording for the revised summations. Please feel free to tick the first checkbox here https://github.com/geneontology/go-ontology/issues/12487#issuecomment-251898523 when that bit is done, so I know.
I will comment here as soon as that merge is done, so you know it's good to proceed with the change in the second checkbox, too.

**Comment by paolaroncaglia** [comment]
Hi @KarenRothfels ,
I completed the merge, so you may proceed with the step outlined in the second checkbox here please: 
https://github.com/geneontology/go-ontology/issues/12487#issuecomment-251898523
Many thanks.

**Comment by paolaroncaglia** [comment]
@KarenRothfels, please also add: 
- [ ] Anchoring of the basal body to the plasma membrane - Represents GO Biological Process - GO:0097711 ’ciliary basal body docking’
- [ ] @KarenRothfels, please add: Intraflagellar transport - Represents GO Biological Process - GO:0035735 ‘intraciliary transport involved in cilium assembly’
Thanks!

**Comment by paolaroncaglia** [comment]
Reactome ‘Assembly of the primary cilium’ has the following structure:

- Assembly of the primary cilium
- [ ] Add dbxref reactome entry (Reactome:R-HSA-5617833.2) to ‘cilium assembly’
- [ ] Add link ‘cilium assembly’ starts_with ’ciliary basal body docking’
- [ ] Add links: GO:1905349 ‘ciliary transition zone assembly’ part_of ‘cilium assembly’, and inverse has_part

- - Anchoring of the basal body to the plasma membrane
(has no subprocesses, just events)
Corresponds to GO:0097711 ’ciliary basal body docking’ and is already referenced.
- [ ] add ‘anchoring of the basal body to the plasma membrane’ as synonym of ’ciliary basal body docking’
- [ ] Can we add part_of children to ’ciliary basal body docking’, among existing GO terms, based on the Reactome view? E.g. ‘NPHP complex’ capable_of_part_of ’ciliary basal body docking’?
- [ ] Add link ‘cilium assembly’ has_part ‘vesicle targeting, trans-Golgi to periciliary membrane compartment’

- - Cargo trafficking to the periciliary membrane
(has subprocesses, but their specificity looks out of scope for GO)
- - - VxPx cargo-targeting to cilium  
- - - BBSome-mediated cargo-targeting to cilium
- - - Trafficking of myristoylated proteins to the cilium
- - - ARL13B-mediated ciliary trafficking of INPP5E
- [ ] Add link ‘protein localization to cilium’ part_of ‘cilium assembly’? When proteins are transported to (or maintained in) a cilium, does this always and only occur when the cilium is assembled? Or can it happen, e.g., during cilium disassembly? 
- [ ] Can we add other links from existing GO terms, based on the Reactome view? E.g. ‘BBSome’ capable_of_part_of ‘cilium assembly’?

- - Intraflagellar transport
(has no subprocesses, just events)
Corresponds to GO:0035735 ‘intraciliary transport involved in cilium assembly’.
- [ ] Add link: ‘cilium assembly’ has_part ‘intraciliary transport involved in cilium assembly’
- [ ] Add ‘intraflagellar transport’ as a broad synonym of ‘intraciliary transport involved in cilium assembly’.
- [ ] Add Reactome:R-HSA-5620924.2 as a dbxref to ‘intraciliary transport involved in cilium assembly’.
- [ ] Can we add part_of children to ’intraciliary transport involved in cilium assembly’, among existing GO terms, based on the Reactome view? E.g. ‘intraciliary transport particle A’ (and B) capable_of_part_of…

- [ ] Scan all existing terms that contain text string ‘cili’ in the BP branch to see if I’ve forgotten any.

**Comment by paolaroncaglia** [comment]
- [ ] For me to check: any more links I can add based on the Reactome summation?
"The primary cilium is one of two main types of cilia present on the surface of many eukaryotic cells (reviewed in Flieghauf et al, 2007). Unlike the motile cilia, which are generally present in large numbers on epithelial cells and are responsible for sensory function as well as wave-like beating motions, the primary cilium is a non-motile sensory organelle with roles in signaling and development and is present in a single copy at the apical surface of most quiescent cells (reviewed in Hsiao et al, 2012). Cilium biogenesis involves the anchoring of the basal body, a centriole-derived organelle, near the plasma membrane and the subsequent polymerization of the microtubule-based axoneme and extension of the plasma membrane (reviewed in Ishikawa and Marshall, 2011; Reiter et al, 2012). Although the ciliary membrane is continuous with the plasma membrane, the protein and lipid content of the cilium and the ciliary membrane are distinct from those of the bulk cytoplasm and plasma membrane (reviewed in Emmer et al, 2010; Rohatgi and Snell, 2010). This specialized compartment is established and maintained during cilium biogenesis by the formation of a ciliary transition zone, a proteinaceous structure that, with the transition fibres, anchors the basal body to the plasma membrane and acts as a ciliary pore to limit free diffusion from the cytosol to the cilium (reviewed in Nachury et al, 2010; Reiter et al, 2012). Ciliary components are targeted from the secretory system to the ciliary base and subsequently transported to the ciliary tip, where extension of the axoneme occurs, by a motor-driven process called intraflagellar transport (IFT). Anterograde transport of cargo from the ciliary base to the tip of the cilium requires kinesin-2 type motors, while the dynein-2 motor is required for retrograde transport back to the ciliary base. In addition, both anterograde and retrograde transport depend on the IFT complex, a multiprotein assembly consisting of two subcomplexes, IFT A and IFT B. The primary cilium is a dynamic structure that undergoes continuous steady-state turnover of tubulin at the tip; as a consequence, the IFT machinery is required for cilium maintenance as well as biogenesis (reviewed in Bhogaraju et al, 2013; Hsiao et al, 2012; Li et al, 2012; Taschner et al, 2012; Sung and Leroux, 2013). The importance of the primary cilium in signaling and cell biology is highlighted by the wide range of defects and disorders, collectively known as ciliopathies, that arise as the result of mutations in genes encoding components of the ciliary machinery (reviewed in Goetz and Anderson, 2010; Madhivanan and Aguilar, 2014)."

**Comment by KarenRothfels** [comment]
ignorant question - how/where do I click the tick boxes? Being logged in here is not sufficient.

update:

-pathway name changed to Cilium Assembly
-3 Go Terms added ( GO:0042384 cilium assembly , GO:0097711 ’ciliary basal body docking, GO:0035735 ‘intraciliary transport involved in cilium assembly).
-removed the word 'primary' from summations, where that quick fix was appropriate and sufficient.  Have three still to tweak that will require a more thorough reworking

**Comment by paolaroncaglia** [comment]
Hi @KarenRothfels ,
Yours is not a silly question at all… I believe that only the ‘author’ of the tick boxes can check or uncheck them. Which would make sense.
So I did that for you. Thanks for updating Reactome! I understand that those changes will be visible in your next release in December, as you said previously. 
Should we have any more updates to suggest for you to do, I’ll let you know here. Thanks and have a good day!

**Comment by paolaroncaglia** [comment]
Added the following links:
cilium assembly has_part axoneme assembly
intraciliary transport particle capable_of_part_of intraciliary transport involved in cilium assembly

**Comment by paolaroncaglia** [comment]
Hi @krchristie and @JohnvanDam 
A question I hope you may help me with please.
I’m wondering if we need to add links between ‘protein localization to cilium’ and ‘cilium assembly’
(former part_of latter, and/or latter has_part former).
When proteins are transported to (or maintained in) a cilium, does this always and only occur when the cilium is assembled? Or can it happen, e.g., during cilium disassembly or maintenance? Thanks.

**Comment by JohnvanDam** [comment]
It is not exclusively happening during cilium assembly. During cilium maintenance and normal cilium operation proteins are constantly being transported to and from the cilium. A nice example is the massive amounts of rhodopsins that are being recycled in photoreceptors. Tubulin is also constantly being imported and exported.

**Comment by paolaroncaglia** [comment]
@JohnvanDam 
Thanks for confirming. So we should have 
cilium assembly has_part protein localization to cilium
because if proteins aren't transported to the cilium and/or kept there, the cilium can't assemble,
but we cannot have 
protein localization to cilium part_of cilium assembly.
If that sounds right, I'll implement.

**Comment by JohnvanDam** [comment]
That sounds right to me

**Comment by krchristie** [comment]
sounds right to me too.

**Comment by paolaroncaglia** [comment]
Note for self: for this project, we’ve worked on the ‘cilium assembly’ branch.
- [ ] Future work: we may want to look at other children of ‘cilium organization’, namely ‘cilium disassembly’ and (not existing yet) ‘cilium maintenance’.

**Comment by doughowe** [comment]
Hi All,
   The GO ontology file we are loading is failing because the current "motile cilium assembly" term is using as it's primary ID an ID that has formerly been a secondary ID for the same term.  Its as if a term merge was done term 1-> term 2 and then the decision was made at a later date to instead merge term 2 -> term 1?  That has the effect of creating a term (Currently GO:0044458) using an ID that was once logged as a secondary term ID, and hence it is no longer valid as a primary ID....  I've included the two term stanzas we are seeing below, one from Oct 13 and one from Oct 26.

I wanted to check in if this was an intended change or a bug or mistake?  We can make updates in our database to permit this if necessary, but it does cause a failure of our GO load due to the violation of reusing a secondary ID as a primary ID.

Thanks for your input on the situation!

Doug

file on Oct 13 2016

data-version: releases/2016-10-08
date: 07:10:2016 10:10

[Term]
id: GO:1903887
name: motile cilium assembly
namespace: biological_process
alt_id: GO:0044458
def: "The aggregation, arrangement and bonding together of a set of components to form a motile cilium." [GO_REF:0000079, GOC:cilia, GOC:krc, GOC:TermGenie, PMID:19776033, PMID:21129373, ZFIN:dsf]
synonym: "motile primary cilia assembly" RELATED [GOC:TermGenie]
synonym: "motile primary cilia formation" RELATED [GOC:TermGenie]
synonym: "motile primary cilium assembly" RELATED []
synonym: "motile primary cilium formation" RELATED [GOC:TermGenie]
synonym: "nodal cilium assembly" RELATED [GOC:TermGenie]
synonym: "nodal cilium formation" RELATED [GOC:TermGenie]
is_a: GO:0042384 ! cilium assembly
created_by: krc
creation_date: 2015-02-06T17:23:23Z

***********************************

file on Oct 26

data-version: releases/2016-10-26
date: 25:10:2016 17:46

[Term]
id: GO:0044458
name: motile cilium assembly
namespace: biological_process
alt_id: GO:1903887
def: "The aggregation, arrangement and bonding together of a set of components to form a motile cilium." [GO_REF:0000079, GOC:cilia, GOC:krc, GOC:TermGenie, PMID:19776033, PMID:21129373, ZFIN:dsf]
synonym: "motile primary cilia assembly" RELATED [GOC:TermGenie]
synonym: "motile primary cilia formation" RELATED [GOC:TermGenie]
synonym: "motile primary cilium assembly" RELATED []
synonym: "motile primary cilium formation" RELATED [GOC:TermGenie]
synonym: "nodal cilium assembly" RELATED [GOC:TermGenie]
synonym: "nodal cilium formation" RELATED [GOC:TermGenie]
is_a: GO:0060271 ! cilium assembly
created_by: krc
creation_date: 2015-02-06T17:23:23Z

**Comment by paolaroncaglia** [comment]
Hi @doughowe ,
Thanks for reporting this. I'm moving discussion to https://github.com/geneontology/go-ontology/issues/12765.

**Comment by paolaroncaglia** [comment]
@krchristie 
As you suggested previously, could you please ask your colleague from the Chlamydomonas community to take a look at the revised ‘cilium assembly’ branch in GO, and let us know if we should restrict any terms to specific taxonomic groups. Currently, ‘cilium assembly’ is labelled as occurring only in eukaryotes but never in Ascomycetes; we could add more constraints if needed, to it and/or to any of its children. Or we could leave everything as is :-) Thanks!

**Comment by paolaroncaglia** [comment]
@KarenRothfels , @deustp01 
All revisions in the ‘cilium assembly’ branch in GO should now have percolated to GO browsers. Could you please take a look at the final structure, and let me know if you approve and/or if you have any final comment. Many thanks again for working with us on this!

**Comment by paolaroncaglia** [comment]
Note for self: once Karen C and Karen R/Peter have replied to my two comments above, I can close this ticket. I'll set to pending while I wait to hear from them.

**Comment by krchristie** [comment]
@paolaroncaglia 
I'm happy to check with experts on whether there are additional taxonomic constraints that would be appropriate for cilium assembly, however my understanding is that Greg Pazour and some of the other researchers that he pulled in to help address our questions on cilium assembly have thought more about these evolutionary questions, though we can certainly include Pete Lefebvre (Chlamydomonas researcher) also. If that sounds good to you, I can send an email to all of them.

**Comment by paolaroncaglia** [comment]
@krchristie 
That would be great, thank you very much in advance!

**Comment by paolaroncaglia** [comment]
Note for self: Karen emailed the experts today.

**Comment by KarenRothfels** [comment]
@paolaroncaglia 
The revisions in the ‘cilium assembly’ branch in GO look good to me.
thanks, 
Karen

**Comment by paolaroncaglia** [comment]
@KarenRothfels 
Great. Thanks again for your help!

**Comment by paolaroncaglia** [comment]
@krchristie 
I assume you didn't hear back from the experts about any need to add further taxon constraints to ciliary terms. I will therefore consider this matter dealt with for now, and of course we can always go back and add taxon rules if and when the need emerges. I'm closing this ticket. Thanks again!

**Comment by krchristie** [comment]
No, I didn't hear back from any of them. I also took a quick look last week at some papers discussing evolutionary relationships of ciliary proteins and didn't think it would be quick or easy to decide if there are additional taxa that never have cilia, so sounds good to leave it as is unless we get additional info.

**Comment by paolaroncaglia** [comment]
Thanks @krchristie .

**Comment by paolaroncaglia** [comment]
Reopening because Peter Satir replied:

“My suggestion is to look at the classification of living organisms which is Appendix 2 in my version of Campbell’s Biology, then Google the phylum and “cilia”. In plants, there are no cilia in Angiosperms and conifers –but in a slightly different phylum - Gingko (a common city tree) has motile ciliated sperm as do mosses, bryophytes etc. Among the protists and fungi there are probably other phyla where cilia are absent (slime molds, for example) but I am familiar only with the typical examples (e.g. Dictyostelium, Physarum). I think all animal phyla have either motile or sensory cilia or both.”

Currently, ‘cilium’ and ‘cilium assembly’ are restricted to
relationship: never_in_taxon NCBITaxon:4890 ! Ascomycota
relationship: only_in_taxon NCBITaxon:2759 ! Eukaryota

To keep this simple, I’d stick to 

- [ ] “In plants, there are no cilia in Angiosperms and conifers”
This should mean adding the same *never_in* taxon rules to ‘cilium’ and ‘cilium assembly’ as in https://github.com/geneontology/go-ontology/issues/12672#issuecomment-251089425, where Tanya advised:

never in Magnoliophyta (NCBITaxon:3398)
never in Zygnematales (NCBITaxon:3176)
never in Gnetales (NCBITaxon:3378)
never in Coniferophyta (NCBITaxon:3312)

“The evolution of land plant cilia
http://onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.2012.04197.x/full (PMID:22691130)
See Figure 3 and accompanying legend, "Land plant phylogeny and the loss of cilia. (a) Cladogram showing the relationship between Chlorophyte, Charophyte (vertical black bar) and land plant (vertical green bar) groups. In both the Charophytes and land plants, where produced, cilia are restricted to sperm cells. The most parsimonious explanation for the distribution of cilia in land plants is two loss events (red stars): once at the base of the Angiosperms and once in the Gymnosperms, where Ginkgo and the cycads diverge from the nonciliated conifers and Gnetales.””

- [ ] “Among the protists and fungi there are probably other phyla where cilia are absent (slime molds, for example) but I am familiar only with the typical examples (e.g. Dictyostelium, Physarum).”

PMID: 16824949 says “Having arisen early in eukaryotic evolution, cilia can be found across a broad phylogenetic spectrum with certain exceptions including Cyanidioschyzon, Arabidopsis, Dictyostelium, and Saccharomyces (Cavalier-Smith, 2002). Although cilia are only found in select cell types in invertebrates, such as the dendritic ends of sensory neurons of Caenorhabditis or in the sperm and sensory neurons of Drosophila, they are present almost ubiquitously in vertebrate cells.”
Cavalier-Smith, 2002 = PMID: 11931142

So I’ll add ‘cilium’ and ‘cilium assembly’ never_in Dictyostelium.
